### ISSUE TITLE
Local.CollectionChanged gets raised twice for the same operation

### ISSUE THREAD

**Issue by Wain123** [aspnet/EntityFramework#9062]
```
Blog b = ctx.Blogs.First();
ctx.Blogs.Local.CollectionChanged += (sender, e) => { Console.WriteLine("changed"); };
ctx.Blogs.Local.Remove(b);
```
This outputs "changed" twice.

**Comment by ajcvickers** [comment]
@Wain123 What version of EF Core are you using? This is fixed in the current dev branch, so unless you're seeing this on the currently nightly builds I'll close as fixed.

**Comment by Wain123** [comment]
1.1.2.0
If it's fixed in 2.0, feel free to close. Sorry.
